account_invoice_customer_no_autofollow/models/account_move.py

```diff
@@ -0,0 +1,31 @@
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def message_subscribe(self, partner_ids=None, channel_ids=None, subtype_ids=None):
+        if self.env.context.get("no_auto_follow"):
+            partner_ids.remove(self.partner_id.id)
+        return super(AccountMove, self).message_subscribe(
+            partner_ids, channel_ids, subtype_ids
+        )
+
+    @api.model_create_multi
+    def create(self, values):
+        self.with_context(
```

Comment:
You have no record here yet. Calling anything is pointless.

Reply:
fixed